src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -223,36 +244,82 @@ private function fixFunction(Tokens $tokens, int $startFunctionIndex): bool
             }
         }
 
+        $argumentCount = 1;
```

Comment:
Why starting at 1? Shouldn't it be 0?

Please add a test case with a function with a newline between parentheses but with 0 arguments. Even with the new option, the newline should be removed, e.g.
```diff
-foo(
-);
+foo();
```